src/UI/SlotArea.cpp

```diff
@@ -2338,7 +2575,3 @@ cItem * cSlotAreaTemporary::GetPlayerSlots(cPlayer & a_Player)
 	}
 	return itr->second.data();
 }
-
-
-
-
```

Comment:
It makes MSVS jump around the file when editing near the end and starting the debug session